scripts/get_device_login_key.py

```diff
@@ -26,27 +26,30 @@
 _examples = """example usage:
 
 python get_device_login_key.py -i "111.222.11.22" -p 10002\n
+
 """  # noqa E501
 
 parser = ArgumentParser(
-    description="Discover and print info of Switcher devices",
+    description="Get the login key of your Switcher device",
     epilog=_examples,
     formatter_class=RawDescriptionHelpFormatter,
 )
+parser.add_argument(
+    "-i",
+    "--ip-address",
+    required=True,
+    help="the ip address assigned to the device",
+    type=str,
+)
+possible_ports = [10002, 20002, 10003, 20003]
```

Comment:
@YogevBokobza please try to avoid changes that are not requested during the review 
https://github.com/TomerFi/aioswitcher/discussions/715#discussioncomment-9018883 I had no comments from my side (PR was approved) and now we go step back. This can be easily added in a new PR after this one was approved. 

Please use port constants from https://github.com/TomerFi/aioswitcher/blob/acc2d035383f8998feffd5409fe9e0adfd6f75d0/src/aioswitcher/bridge.py#L48-L53

Reply:
@thecode I am trying..
Anyway, what you asked for is done.